docs/collab/133.md

```diff
@@ -0,0 +1,28 @@
+# 5	Weitere Open-Science-Themen
+## 5.1 Berücksichtigung von Open Source in Förderinstrumenten
+
+Im Rahmen der in den letzten zwei Jahren etablierten [Community of Practice "Open Source"](https://bfh.github.io/contact#bfhchopensource) haben wir aus Sicht (AHB, HAFL, TI, W und Vizerektorat) erste strategische [Anlaufpunkte zum Thema Open Source](http://bfh.ch/opensource) an der BFH aufbauen können.
+
+Analog zu 4.1 stellen auch wir fest, dass die "Mitarbeit an" und "Freigabe von" quelloffenen Systemen bisher nicht als eigenständige (und gleichwertige) Leistung in der Forschungsinitiative berücksichtigt wird (s. [#52](https://github.com/bfh/opensource/issues/52)).
+
+Nebst einem **klaren Committment zur präferierten ([open-by-default](https://en.wikipedia.org/wiki/Open_by_default)) [Nutzung (1), Mitarbeit (2) & Freigabe (3)](https://bfh.github.io) von Artefakten** unter freien und offenen [Lizenzen nach SPDX](https://spdx.org/licenses/): 
+   - für Software: [Open Source Initiative (OSI)](https://opensource.org/license),
+   - für Dokumentation & kreative Inhalte: [Creative Commons (CC)](https://creativecommons.org/cclicenses/),
+   - sowie für (offenen) Daten, KIs und Hardware
+
+in öffentlichen, frei zugänglichen, nicht kommerziellen Repositorien (wie z.B. [github.com/bfh](https://github.com/bfh)), schlagen wir vor explizit zu Fördern und zu Honorieren
+ - (1) Open-Source-Artefakte zu **Nutzen** u.a. 
+   - bei der (Neu)-Beschaffung und -Aufbau von Lösungen und Systemen
+   - bei der Evaluation von (offenen) Alternativen und dem Wechsel zu der Bestandslösungen
+   - ...
+ - (2) das **Mitwirken an bestehenden Open-Source-Projekten**, die im Rahmen von Initiativen beispielsweise in Form von Frameworks, Bibliotheken und ganzen Softwareprojekten zum Einsatz kommen durch z.B.
+   - das Melden von Fehlern & Anforderungslücken 
+   - dem Beitragen der Fehlerbehebungen & Anfoderungslücken
+   - der Mitarbeit an Dokumentations- und Beispiellücken
+   - das Betreuen von Communities und deren Prozessen
+   - ...
+ - (3) das **Initiieren von neuen und Erhalten von bestehenden Projekten**
+   - Durchführen von Marktrecherchen zu ggfs. bereits bestehenden offenen Alternativprojekten
+   - Vorbereitung und Begleitung der Lancierung eines Vorhabens
+   - kontinuierliche Unterstützung von als strategisch eingestufen Projekten 
+   - ...
```

Comment:
Könnte hier als Anregung hinzugefügt werden, dass wir für (2) und (3) ein zentrales Förderkonto und entsprechende Planung in den jeweiligen ILVs vorschlagen?

Reply:
See [62edee1](https://github.com/bfh/opensource/commit/62edee1d94bdac651562e4e1e348c1d0317d0f02)